pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -685,7 +685,12 @@ func (l *LibvirtDomainManager) FinalizeVirtualMachineMigration(vmi *v1.VirtualMa
 // hotPlugHostDevices attach host-devices to running domain
 // Currently only SRIOV host-devices are supported
 func hotPlugHostDevices(vmi *v1.VirtualMachineInstance, domain cli.VirDomain) error {
-	sriovHostDevices, err := sriov.CreateHostDevices(vmi)
+	domainSpec, err := util.GetDomainSpecWithFlags(domain, 0)
+	if err != nil {
+		return err
+	}
+
+	sriovHostDevices, err := sriov.GetHostDevicesToAttach(vmi, *domainSpec)
```

Comment:
I tried to minimize the side affect of moving a pointer instead of a struct, but I guess that since I pass the VMI pointer its kinda misses the point..

Reply:
I change it to get pointer now